server/routes/datalogs.js

```diff
@@ -4,11 +4,35 @@ const { NotFoundError } = require("../utils/error");
 var Parse = require("parse/node");
 const BACK4APPKEY = require("../../client/src/securitykeys").BACK4APPKEY;
 const BACK4APPSECRET = require("../../client/src/securitykeys").BACK4APPSECRET;
+const axios = require("axios");
+const got = require("got");
 const fs = require("fs");
+const FormData = require("form-data");
 
 Parse.initialize(BACK4APPKEY, BACK4APPSECRET);
 Parse.serverURL = "https://parseapi.back4app.com/";
 
+// save photo
+router.post("/savePhoto", async (req, res) => {
+  if (!req.body.photo) {
+    return res.status(400).send("No photo");
+  }
+  const { photo } = req.body;
+
+  const { id } = req.body;
+
+  const photoObject = new Parse.Object("Photo");
+
+  let drainPhoto = new Parse.File("photo.jpg", { base64: photo });
```

Comment:
Can this not be const?

Reply:
Yes it can be I can change this

Reply:
Also I believe the added files are from got, which was a package I needed for using the color extraction API.